packages/react-components/react-search/stories/SearchBox/SearchBoxAppearance.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { SearchBox } from '@fluentui/react-search';
+import { makeStyles, mergeClasses, shorthands, tokens, useId, Label } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    maxWidth: '490px',
+  },
+  field: {
+    display: 'grid',
+    gridRowGap: tokens.spacingVerticalXXS,
+    marginTop: tokens.spacingVerticalMNudge,
+    ...shorthands.padding(tokens.spacingHorizontalMNudge),
+  },
+  filledLighter: {
+    backgroundColor: tokens.colorNeutralBackgroundInverted,
+    '> label': {
+      color: tokens.colorNeutralForegroundInverted2,
+    },
+  },
+  filledDarker: {
+    backgroundColor: tokens.colorNeutralBackgroundInverted,
+    '> label': {
+      color: tokens.colorNeutralForegroundInverted2,
+    },
+  },
+});
+
+export const Appearance = () => {
+  const outlineId = useId('searchBox-outline');
+  const underlineId = useId('searchBox-underline');
+  const filledLighterId = useId('searchBox-filledLighter');
+  const filledDarkerId = useId('searchBox-filledDarker');
+  const styles = useStyles();
+
+  return (
+    <div className={styles.base}>
+      <div className={styles.field}>
+        <Label htmlFor={outlineId}>Outline appearance (default)</Label>
+        <SearchBox appearance="outline" id={outlineId} />
+      </div>
```

Comment:
It's better if you use the Field component since it handles the id and label linking out of the box. You can refer to https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-textarea/stories/Textarea/TextareaAppearance.stories.tsx for an example.
```suggestion
      <Field label="Outline appearance (default)">
        <SearchBox appearance="outline" />
      </Field>
```